From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Giovanni <42092549+Matocolotoe@users.noreply.github.com>
Date: Sun, 1 Sep 2024 13:56:40 +0200
Subject: [PATCH] Play old damage animation


diff --git a/src/main/java/net/minecraft/server/level/ServerPlayer.java b/src/main/java/net/minecraft/server/level/ServerPlayer.java
index c396580a9cfd86ff261bed439bb4662ae88010b5..472ee7445f4296c1a3bdf528887256d64015cc64 100644
--- a/src/main/java/net/minecraft/server/level/ServerPlayer.java
+++ b/src/main/java/net/minecraft/server/level/ServerPlayer.java
@@ -2705,7 +2705,7 @@ public class ServerPlayer extends net.minecraft.world.entity.player.Player imple
     @Override
     public void indicateDamage(double deltaX, double deltaZ) {
         this.hurtDir = (float) (Mth.atan2(deltaZ, deltaX) * 57.2957763671875D - (double) this.getYRot());
-        this.connection.send(new ClientboundHurtAnimationPacket(this));
+        this.connection.send(new ClientboundHurtAnimationPacket(0, -180)); // TL - Play old damage animation
     }
 
     @Override
