From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Giovanni <42092549+Matocolotoe@users.noreply.github.com>
Date: Sun, 1 Sep 2024 14:02:42 +0200
Subject: [PATCH] [TL] Nerf critical hit multiplier


diff --git a/src/main/java/net/minecraft/world/entity/player/Player.java b/src/main/java/net/minecraft/world/entity/player/Player.java
index 09bcbc0ae36e4e69fee87a7e0c49acf496117a39..56e9efcb840b201fb6ce476cc804861c3a42505c 100644
--- a/src/main/java/net/minecraft/world/entity/player/Player.java
+++ b/src/main/java/net/minecraft/world/entity/player/Player.java
@@ -1317,7 +1317,7 @@ public abstract class Player extends LivingEntity {
                     flag2 = flag2 && !this.level().paperConfig().entities.behavior.disablePlayerCrits; // Paper - Toggleable player crits
                     if (flag2) {
                         damagesource = damagesource.critical(true); // Paper start - critical damage API
-                        f *= 1.5F;
+                        f *= 1.15F; // TL - Nerf critical hit multiplier
                     }
 
                     float f3 = f + f1;
